impeller/docs/faq.md

```diff
@@ -100,3 +100,133 @@
     as development progresses.
 * How do you run `impeller_unittests` with Playgrounds enabled?
   * Specify the `--enable_playground` command-line option.
+* Describe Impeller in a Tweet.
```

Comment:
Suggest s/Describe Impeller in a Tweet/Describe Impeller in less than 10 words/

Reply:
But thats literally what Seth asked 😄 !

Reply:
hah yes I did, but I like @mit-mit 's suggestion here :)